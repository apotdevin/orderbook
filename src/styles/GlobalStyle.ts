import { createGlobalStyle } from 'styled-components';

export const GlobalStyles = createGlobalStyle`
    html, body {
        margin: 0;
        padding: 0;
    }
    *, *::after, *::before {
        box-sizing: border-box;
    }
    body {
        font-size: 16px;
        background: rgba(0, 0, 0, 0.87);
        color: white;
        font-family: 'Roboto', sans-serif;
        text-rendering: optimizeLegibility;
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
    }
`;
