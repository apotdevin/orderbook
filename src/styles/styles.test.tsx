import { render } from '@testing-library/react';
import { GlobalStyles } from './GlobalStyle';

describe('Global Styles', () => {
  test('are correctly generated in the head', () => {
    render(<GlobalStyles />);
    expect(document.getElementsByTagName('head')).toMatchSnapshot();
  });
});
