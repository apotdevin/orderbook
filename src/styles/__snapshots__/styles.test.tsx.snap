// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Styles are correctly generated in the head 1`] = `
HTMLCollection [
  <head>
    <style
      data-styled="active"
      data-styled-version="5.2.1"
    >
      html,body{margin:0;padding:0;}
      *,*::after,*::before{box-sizing:border-box;}
      body{font-size:16px;background:rgba(0,0,0,0.87);color:white;font-family:'Roboto',sans-serif;text-rendering:optimizeLegibility;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;}
    </style>
  </head>,
]
`;
